pkg/cmd/pulumi-language-java/main.go

```diff
@@ -158,49 +160,55 @@ func (host *javaLanguageHost) Executor(attachDebugger bool) (*executors.JavaExec
 	return executor, nil
 }
 
-// GetRequiredPlugins computes the complete set of anticipated plugins required by a program.
-func (host *javaLanguageHost) GetRequiredPlugins(
+// GetRequiredPackages computes the complete set of anticipated packages required by a program.
+func (host *javaLanguageHost) GetRequiredPackages(
 	ctx context.Context,
-	req *pulumirpc.GetRequiredPluginsRequest,
-) (*pulumirpc.GetRequiredPluginsResponse, error) {
-	logging.V(5).Infof("GetRequiredPlugins: program=%v", req.GetProgram()) //nolint:staticcheck
```

Comment:
Was removing this log intentional?  It still seems useful to know when GetRequirePackages is called (though we are pretty inconsistent with logging overall)

Reply:
It was -- I checked the PR in pulumi/pulumi that added `GetRequiredPackages` for Go, NodeJS and Python, and it seemed like we weren't doing any logging there really for those calls, so I figured I'd remove it. But I can add something back before I merge -- agree it doesn't hurt :+1:

Reply:
Ah fair enough.  I didn't double check for those.  Might be worth adding them there for consistency ;)